Ginger/GingerCore/Platforms/PlatformsInfo/PlatformInfoBase.cs

```diff
@@ -75,6 +75,81 @@ public static PlatformInfoBase GetPlatformImpl(ePlatformType Platform)
                     return null;
             }
         }
+
+        /// <summary>
+        /// This method is used to check if the paltform supports recording
+        /// </summary>
+        /// <param name="Platform"></param>
+        /// <returns></returns>
+        public static bool IsPlatformSupportRecording(ePlatformType Platform)
+        {
+            bool support = false;
+            switch (Platform)
+            {
+                case ePlatformType.AndroidDevice:
+                    support = false;
+                    break;
+
+                case ePlatformType.Java:
+                    support = true;
+                    break;
+
+                case ePlatformType.Web:
+                    support = true;
+                    break;
+
+                case ePlatformType.PowerBuilder:
+                    support = false;
+                    break;
+
+                case ePlatformType.Windows:
+                    support = false;
+                    break;
+
+                case ePlatformType.Mobile:
+                    support = false;
+                    break;
+            }
+            return support;
+        }
+
+        /// <summary>
+        /// This method is used to check if the paltform supports POM
+        /// </summary>
+        /// <param name="Platform"></param>
+        /// <returns></returns>
+        public static bool IsPlatformSupportPOM(ePlatformType Platform)
```

Comment:
this is not correct OO write, in PlatforminfoBase you sholud add abstract IsSupportingPOM and each Platform type will implement the flag true or false, you can make it virtual and return false as default and only in WebPlatform ovveride it with true